### DISCUSSION TITLE
Cloning a repo through an organization

### DISCUSSION THREAD

**Discussion by hillaryAHR** [CollectionBuilder/collectionbuilder.github.io#117] [Q&A] [answered]
Hi! I created a site for a cultural organization with the Collection Builder template, using an account that I set up: https://deepsouthdh.github.io/blmoor/. The people I worked with to create the site want to have better control of the content, and they want the URL to be just blmoor.github.io. So I created an organization in github (https://github.com/blmoor), cloned the repo from the original site, and tried to deploy it as https://blmoor.github.io/, but I get a 404 error. I don't think I changed anything, so I'm wondering if anyone with more github prowess than I can help me figure out why it isn't working, and how to fix it! Thank you!

**Comment by evanwill** [comment, marked as answer]
@hillaryAHR first, the `url` variables in "_config.yml" are off--> you have the project in the repository named "blmoor.github.io", which github pages will serve at `https://blmoor.github.io/` (as you mentioned). 
In "_config.yml" in the URL VARIABLES section currently looks like:

```
url: https://deepsouthdh.github.io/blmoor 
# baseurl is path to location on the domain if necessary e.g. /digital/hjccc
# on GitHub Pages it is your github repository's name prefixed with a /
#baseurl: /collectionbuilder-gh 
# location of code, the full url to your github repository
source-code: https://github.com/DeepSouthDH/blmoor
```

You will want to remove the /blmoor from your url (and I would update the source-code location too), so it will look like: 

```
url: https://deepsouthdh.github.io
# baseurl is path to location on the domain if necessary e.g. /digital/hjccc
# on GitHub Pages it is your github repository's name prefixed with a /
#baseurl: /collectionbuilder-gh 
# location of code, the full url to your github repository
source-code: https://github.com/blmoor/blmoor.github.io/
```

Then it should build correctly. 
However, that shouldn't really cause the 404 page that you are seeing (with the wrong url, just some of the links should be broken and the style sheet missing, but not 404!)... 
so I think I would turn off github pages, and then reactivate it (got to Settings > Pages > under Branch select "none" then Save. Then in little while, go back to Settings > Pages > under Branch select "main" then Save) 
I wonder if something happened during the transfer that it isn't building in the correct location? 

Let us know if it works!

**Reply by hillaryAHR** [reply]
Updating the yaml file worked! Thank you. I started to turn off the page settings as you suggested, but tested the link first and it worked without doing that. I'm not sure why it gave me the 404, but it's working now. Thanks again!
